lib/glueby/internal/wallet/tapyrus_core_wallet_adapter.rb

```diff
@@ -107,7 +107,7 @@ def sign_tx(wallet_id, tx, prevtxs = [])
         end
 
         def broadcast(wallet_id, tx)
-          client.sendrawtransaction(tx.to_hex)
+          RPC.client.sendrawtransaction(tx.to_hex)
         end
```

Comment:
This method doesn't seem to need to take `wallet_id` as an argument. Also, if you are passing a `wallet_id`, it would be correct to say `perform_as (wallet_id) ..` instead of `RPC.client ...`.

@Yamaguchi what do you think?

Reply:
I prefer to  use `perform_as(wallet_id)` than RPC.client directly.

Reply:
@JunOkumura Can you change to use `perform_as(wallet_id)` ?

Reply:
ok. I try it soon.